src/server/auth/server/roles.go

```diff
@@ -183,6 +183,7 @@ func init() {
 				auth.Permission_CLUSTER_ENTERPRISE_GET_CODE,
 				auth.Permission_CLUSTER_ENTERPRISE_DEACTIVATE,
 				auth.Permission_CLUSTER_DELETE_ALL,
+				auth.Permission_PIPELINE_GET_LOGS,
```

Comment:
Also can we make a role to go with this permission so users don't have to be all-or-nothing admins?